### ISSUE TITLE
Create a new skill to airdrop a quadbike (similar to paratroopers)

### ISSUE THREAD

**Issue by fritogotlayed** [DUWS-R-Team/DUWS-R#43]
Create a new radio skill similar to the paratroopers skill that air lifts a quad-bike in for the player. CP cost should probably be 3 for now but this could be adjusted in the future during play testing.

**Comment by rlex** [comment]
Whole point of skills is that they're bound to player and doesn't require any CP to use, rather using cooldown to balance it. It probably should have pretty big (30m-1h?) cooldown rather than CP cost

**Comment by fritogotlayed** [comment]
Skill was probably the wrong term to use then.

**Comment by fritogotlayed** [comment]
@rlex I just updated the description to use the term "support" rather than "skill" to remove the confusion.

**Comment by rlex** [comment]
Yeah, also you used word "radio". Radio menu in DUWS = player skills.

**Comment by fritogotlayed** [comment]
Updated to remove the word "radio" as well.
